### ISSUE TITLE
fuzzy search (typo tolerance) support

### ISSUE THREAD

**Issue by ayZagen** [CloudCannon/pagefind#756]
It would be great to have configurable fuzzy search support like meilisearch

https://www.meilisearch.com/docs/learn/relevancy/typo_tolerance_settings#enabled

**Comment by bglw** [comment]
Agreed! It's a bit of a harder bar for Pagefind due to the bandwidth-restricted index structure, but there might be some interesting ways to tackle that.

Best place to follow updates is the #2 issue :)
